grug/storage/src/codec.rs

```diff
@@ -17,6 +17,7 @@ pub trait Codec<T> {
 // ----------------------------------- borsh -----------------------------------
 
 /// Represents the Borsh encoding scheme.
+#[derive(Clone)]
 pub struct Borsh;
```

Comment:
Might as well derive all common traits (debug, clone, copy, partialeq, eq) for all other `Codec`'s as well